module_info_import/models/module_information.py

```diff
@@ -1,43 +1,40 @@
-import requests
-from odoo import models, api, release, _
-from odoo.exceptions import UserError
 import logging
 
+import requests
+import yaml
+
+from odoo import _, api, models
 
 _logger = logging.getLogger(__name__)
 
 
 ERROR_MESSAGE = _("There is an issue with module information synchronization")
+VERSIONS = ["14.0", "15.0", "16.0"]
 
 
 class ModuleInformation(models.Model):
     _inherit = "module.information"
 
+    @api.model
+    def get_module_info(self, version):
+        # make it multi version 14.0 /15.0 /16.0
+        url = (
+            "https://raw.githubusercontent.com/akretion"
+            f"/odoo-module-tracker/gh-pages/{version}.yml"
+        )
+        response = requests.get(url)
+        modules_list = yaml.safe_load(response.text)
+        return modules_list
```

Comment:
direct return to avoid to define a variable
```suggestion
        return yaml.safe_load(response.text)
```